python/ray/serve/examples/doc/tutorial_tensorflow_full.py

```diff
@@ -0,0 +1,70 @@
+# This file is a copy of python/ray/serve/examples/doc/tutorial_tensorflow.py. If 
+# you make any changes, please make the changes to both files.
+
+# fmt: off
```

Comment:
For all the full scripts:

1. Include the imports in the `__doc_full_begin__` section, so they show up in the documentation. This may require removing the `fmt: off` and `fmt: on` comments, but I think this should be okay.
2. Add a comment mentioning the file name at the top of the code that the user reads. E.g. in this file, we could add:

```python
# File name: tutorial_tensorflow.py
from ray import serve
...
```

This helps users understand what the command `serve run file_name:app` refers to.